test/integration/running/test_running.py

```diff
@@ -442,20 +442,17 @@ def test_running_reread_config(tt_cmd, tmpdir):
     # Wait for child process of instance to start.
     # We need to wait because watchdog starts first and only after that
     # instances starts. It is indicated by 'started' in logs.
-    log_file_path = os.path.join(tmpdir, "test_app", log_path, "test_app", "tt.log")
     file = wait_file(os.path.join(tmpdir, "test_app", log_path, "test_app"), 'tt.log', [])
     assert file != ""
-    isStarted = wait_instance_start(log_file_path)
-    assert isStarted is True
+    file = wait_file(os.path.join(tmpdir, "test_app", run_path, "test_app"), pid_file, [])
```

Comment:
`pid_file` is a `tt.pid`, which is created by tt and does not mean that tarantool if up. So we cannot rely on this.

Reply:
Changed